english/covid-19/0/0-BB.md

```diff
@@ -1,7 +1,7 @@
-Great! Which game will you play?
+Great! Which game 🎲 will you play?
 
-[Rock Paper Scissor](0-BBA.md)
+[🪨📄✂️ Rock Paper Scissor](0-BBA.md)
```

Comment:
What is this sign mean?

Reply:
These are just emojis symbolizing rock-paper-scissors.

Reply:
I'm only could see the question mark sign

Reply:
Okay change it to the emojis that you have